src/panels/config/config-entries/ha-ce-entities-card.js

```diff
@@ -19,7 +19,7 @@ function computeEntityName(hass, entity) {
 /*
  * @appliesMixin EventsMixin
  */
-class HaDeviceCard extends EventsMixin(PolymerElement) {
+class HaCeEntitiesCard extends EventsMixin(PolymerElement) {
```

Comment:
We forgot to apply the localize mix in here.

Fix here: https://github.com/home-assistant/home-assistant-polymer/pull/1757

Now i need to consider what I want to do for 0.80 😢

Reply:
(verified that this was the only one in this PR)